java-analyzer-bundle.test/docs/README.md

```
@@ -0,0 +1,325 @@
+podman build -t jdtls-analyzer:test ..
+
+# Run tests in container
+cd ../integration
+./run_local.sh
+```
+
+---
+
+## 📖 Document Relationships
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)